source/main.ts

```diff
@@ -74,29 +74,47 @@ const browsers: BrowserManifests = {
 if (args._[0] === 'chrome') delete browsers.firefox
 if (args._[0] === 'firefox') delete browsers.chrome
 
-const entryPoints = [
-  'options.tsx',
-  'content_script.ts',
-  'background.ts',
-  'popup.tsx',
-].map((file) => `${args.source}/${file}`)
+const entryNames = [
+  'options',
+  'content_script',
+  'contentScript',
+  'background',
+  'popup',
+]
+const entryPoints: string[] = []
+
+await Promise.all(entryNames.map(async (name) => {
+  const jsPath = `${args.source}/${name}.js`
+  const jsxPath = `${args.source}/${name}.jsx`
+  const tsPath = `${args.source}/${name}.ts`
+  const tsxPath = `${args.source}/${name}.tsx`
+  if (await exists(jsPath)) entryPoints.push(jsPath)
+  else if (await exists(jsxPath)) entryPoints.push(jsxPath)
+  else if (await exists(tsPath)) entryPoints.push(tsPath)
+  else if (await exists(tsxPath)) entryPoints.push(tsxPath)
```

Comment:
```suggestion
  for (const extension of ['js', 'jsx', 'ts', 'tsx']) {
    const path = `${args.source}/${name}.${extension}`
    if (await exists(path)) {
      entryPoints.push(path)
      break
    }
  }
```
I'd put this into a loop instead of writing duplicate code. Might be a bit less readable tho

Reply:
eh, If I thought the list might expand I might have done something like this, but for something this small, I think I prefer the "dumb" solution so I don't have to think at all 😛. I guess it is more prone to typos though.